src/main/resources/openapi/audio-requests.yaml

```diff
@@ -0,0 +1,96 @@
+openapi: 3.0.1
+
+servers:
+  - url: http://localhost:4550/
+
+info:
+  description: Modernised DARTS (Digital Audio Recording and Transcription Service).
+  version: "0.0.1"
+  title: Modernised DARTS
+
+paths:
+  /audio-requests:
+    get:
+      security:
+        - bearerAuth: [ ]
+      tags:
+        - Audio Requests
+      summary: Users can view a summary of their audio requests and status
+      operationId: getYourAudio
+      description: '"Your audio" requests for a user to view whether Current / Expired'
+      parameters:
+        - in: header
+          name: user_id
+          schema:
+            type: integer
+          description: The user_account primary key
+          required: true
+        - in: query
+          name: expired
+          schema:
+            type: boolean
+          description: A way to filter whether Current / Expired
+          required: true
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/AudioRequestSummary'
+        '500':
+          description: Internal Server Error
+          content:
+            application/json+problem:
+              schema:
+                $ref: './problem.yaml'
+
+components:
+  securitySchemes:
+    bearerAuth: # arbitrary name for the security scheme
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+
+  schemas:
+    AudioRequestSummary:
+      type: object
+      properties:
+        media_request_id:
+          type: integer
+          example: 12345
+        case_number:
+          type: string
+          example: "T20200190"
+        courthouse_name:
+          type: string
+          example: "Manchester Minshull Street"
+        hearing_date:
+          type: string
+          format: date
+          example: "2023-08-17"
+        media_request_start_ts:
+          type: string
+          format: date-time
+          example: "2023-08-21T09:00:00Z"
+        media_request_end_ts:
+          type: string
+          format: date-time
+          example: "2023-08-21T10:00:00Z"
+        media_request_expiry_ts:
+          type: string
+          format: date-time
+          example: "2023-08-23T09:00:00Z"
+        media_request_status:
+          $ref: '#/components/schemas/MediaRequestStatus'
+
+    MediaRequestStatus:
+      type: string
+      enum:
+        - OPEN
+        - PROCESSING
+        - FAILED
+        - COMPLETED
+        - EXPIRED
```

Comment:
The wireframes seem to have a status called "Ready"
https://www.figma.com/file/sjyyNfzv1pFupjMuVKJXID/DARTS-(Build)?type=design&node-id=1284-20324&mode=design&t=zxQmo7M6YxTbu1HR-0

Reply:
the state model has "Requested"
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1672094567#AudioTransformationServiceLLD-Statemodel

Reply:
I raised this during refinement / estimation and during the sprint development - I've updated the JIRA description / swagger to document the media_request_status string Enum may be one of [ OPEN, PROCESSING, FAILED, COMPLETED ] as Current View and added the new status EXPIRED also.
FYI @hemantasharma1129